restore-with-rid/test.json

```diff
@@ -7,5 +7,6 @@
   "type": "bash",
   "cleanup": true,
   "ignoredRIDs":[
+    "linux-s390x"
```

Comment:
This test does a:

     dotnet restore -r rhel.8-s390x

Which fails with:

```
restore-with-rid.csproj : error NU1101: Unable to find package Microsoft.NETCore.App.Runtime.linux-s390x. No packages exist with this id in source(s): [... ]
restore-with-rid.csproj : error NU1101: Unable to find package Microsoft.AspNetCore.App.Runtime.linux-s390x. No packages exist with this id in source(s): [...]
```

Is this expected?

Reply:
Any thoughts @tmds ?

Reply:
> Is this expected?

Yes, the sdk doesn't contain these nupkgs, and nuget.org doesn't have them either.